### ISSUE TITLE
Perfoming way worse than EffnetV1

### ISSUE THREAD

**Issue by macsunmood** [leondgarse/keras_efficientnet_v2#8]
The issue I have is that, for example, with the same config EffnetV2_B0 performs way worse then original EfficentnetB0 from TF.applications (~0.70 acc vs. ~0.90).
Any quick/obvious reason for that or is more detailed info needed?

**Comment by macsunmood** [comment]
Also, I get the following warning at the start of V2 traning, could this be assosiated with the issue?

```
Epoch 1/30

Epoch 00001: LearningRateScheduler reducing learning rate to 0.0010000000474974513.
WARNING:tensorflow:AutoGraph could not transform <function Model.make_train_function.<locals>.train_function at 0x7f772b2fff28> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: closure mismatch, requested ('self', 'step_function'), but source function had ()
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
WARNING: AutoGraph could not transform <function Model.make_train_function.<locals>.train_function at 0x7f772b2fff28> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: closure mismatch, requested ('self', 'step_function'), but source function had ()
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
```

**Comment by macsunmood** [comment]
UPD: I just tried an equivalent model from `https://github.com/sebastian-sz/efficientnet-v2-keras` and it performs just well.
I'm doing transfer learning and using `include_top=False` parameter for it. In `leondgarse/keras_efficientnet_v2` I'm using `num_classes=0` for this purpose.
Still wonder what I has to do with, but looks like there's possibly some bugs with the current implementation to me.

**Comment by leondgarse** [comment]
My bad, it's because I accidentally changed a layer name, makes loading skipped on that layer...
You may try again by upgrade `pip install -U keras-efficientnet-v2`.
Speaking of EfficentnetB0 from `TF.application`, it has a `Rescaling` and `Normalization` layer on the top, so we may use value in `[0, 255]` as inputs, but `[0, 1]` values for `efficientnetV2` models.

**Comment by macsunmood** [comment]
> My bad, it's because I accidentally changed a layer name, makes loading skipped on that layer... Sorry for that..
> You may try again by upgrade `pip install -U keras-efficientnet-v2`.
> Speaking of EfficentnetB0 from `TF.application`, it has a `Rescaling` and `Normalization` layer on the top, so we may use value in `[0, 255]` as inputs, but `[0, 1]` values for `efficientnetV2` models.

The maintainer of the implementation says that, quote: 
> The models expect image values in range -1:+1.
You say, the v2 models expect values in range `[0, 1]`.
Which one is true?

Also, I still have the `WARNING:tensorflow:AutoGraph could not transform <function Model.make_train_function` issue. :( Maybe it's not related to your implementation and has to do with something on my side..

**Comment by leondgarse** [comment]
Yes, yes, it's `[-1, 1]`, he's right! I mad some mistake in my testing, ran `tf.expand_dims(imm / 127 - 127.5, 0)` as input, which results bad, makes me thought it should be `[0, 1]`...
I'm using `TF 2.6.0` in my projects, and didn't meet that `AutoGraph` warning. Maybe version related? Both mine and [sebastian-sz](https://github.com/sebastian-sz/efficientnet-v2-keras) using only typical keras layers, and returns a typical keras model. Maybe the optimizer?
